qpc/messages.py

```diff
@@ -232,7 +233,9 @@
 BECOME_PASSWORD = 'Provide a privilege escalation password to be used when '\
     'running a network scan.'
 
-
+JOB_ID_NOT_FOUND = 'Error finding id %s.'
+JOB_ID_STATUS = 'Job id: %s status is %s'
+DISPLAY_REPORT_ID = 'The report id is %s.'
```

Comment:
Change this message to: 
Created merge report with id: "%s".  To view, run `qpc report summary --report "%s" --csv`